src/components/Address.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+import React from "react";
+
+class Address extends React.Component {
+
+    describe(property: string, description: string): React.Element {
+        var described = "";
+        if (this.props[property]) {
+            described = `${description} ${this.props[property]}`;
+        }
+
+        return <span className={property} >{described}</span>;
+    }
+
+    render(): React.Element {
+        return (
+            <div className="Address">
+                {this.describe('level', 'Level')}
+                {' '}
```

Comment:
multiple consecutive `&nbsp`s will render if there is no content which isn't ideal...